src/status_im2/contexts/shell/bottom_tabs.cljs

```diff
@@ -19,15 +19,21 @@
 
 (defn bottom-tab
   [icon stack-id shared-values notifications-data]
-  [bottom-nav-tab/bottom-nav-tab
-   (assoc (get notifications-data stack-id)
-          :test-ID             stack-id
-          :icon                icon
-          :icon-color-anim     (get
-                                shared-values
-                                (get shell.constants/tabs-icon-color-keywords stack-id))
-          :on-press            #(animation/bottom-tab-on-press stack-id)
-          :accessibility-label (str (name stack-id) "-tab"))])
+  (let [{:keys [key-uid]}   (rf/sub [:multiaccount])
+        customization-color (or (:color (rf/sub [:onboarding-2/profile]))
```

Comment:
why do we have 2 subs? Can't we just have one that works :)

Reply:
As I told you before, this is a pattern existing across the code, so I created an issue to fix it everywhere: #16263 :)